CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+0.1.9 (2020-12-22)
+------------------
+
+- Added the File Upload Widget and SettingsForm E2E testing VRT to validate Overlap for the dialog of Add/Edit options for Customer and User
```

Comment:
The File Upload Widget and the Settings Form are very different features, so please have them on separate lines of the ChangeLog

Reply:
File Upload Widget, SettingsForm VRT are separated.